pkg/odo/cli/pipelines/webhook/delete.go

```diff
@@ -0,0 +1,70 @@
+package webhook
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/spf13/cobra"
+
+	backend "github.com/openshift/odo/pkg/pipelines/webhook"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
+)
+
+const deleteRecommendedCommandName = "delete"
+
+var (
+	deleteExample = ktemplates.Examples(`	# Delete a Git repository webhook 
+	%[1]s`)
+)
+
+type deleteOptions struct {
+	options
+}
+
+func newDeleteOptions() *deleteOptions {
+	return &deleteOptions{}
+}
+
+// Run contains the logic for the odo command
+func (o *deleteOptions) Run() (err error) {
+
+	ids, err := backend.Delete(o.accessToken, o.pipelines, o.getAppServiceNames(), o.isCICD)
+
+	if ids != nil && len(ids) > 0 {
```

Comment:
```suggestion
	if len(ids) > 0 {
```

Reply:
Also, this should probably return early if err != nil.

Reply:
Removed the redundant nil check above.

The reason it does not return early is that we want to report all the deleted IDs in case of partial completion.